scripts/build.py

```diff
@@ -414,6 +413,10 @@ def copy_files_source_to_target(source, target, file_hashes):
 
             target_path = source_path
             relative_path = os.path.relpath(source_path, source)
+            # Remove file that was deleted in DEV.
```

Comment:
I feel like this doesn't follow the contract of the function, as given in the docstring. Let's either update the docstring or move this functionality outside the function.

Reply:
I have added this detail to the docstring. This was added because there is no "cleanup" check for files that was either deleted or renamed in DEV but not yet deleted in staging directory.